playwright/tests/workspaces/workspace.ts

```diff
@@ -3,3 +3,5 @@ import * as path from 'path';
 export const defaultWorkspacePath = path.resolve(__dirname, './default');
 export const noEngineWorkspacePath = path.resolve(__dirname, './no-engine');
 export const noProjectWorkspacePath = path.resolve(__dirname, './no-project');
+
+export const userDialogPID = '15254DCE818AD7A2-f3';
```

Comment:
maybe this should be part of the process editor test or the process editor page object?